utils/assertServerEnv.ts

```diff
@@ -0,0 +1,5 @@
+export function assertServerEnv(functionName: string) {
```

Comment:
FYI there's also `server-only` package that you can import into a module to mark it as such.

Reply:
0fa9f52c8344f91f221d5cc86cfa2e1cf3c808b2